instrument/D2B_Definition.xml

```diff
@@ -14,17 +14,33 @@ name="D2B" valid-from="1900-01-31 23:59:59" valid-to="2100-01-31 23:59:59" last-
       <handedness val="right" />
     </reference-frame>
   </defaults>
-  <component type="moderator">
-    <location z="-2" />
+  <component type="monochromator">
```

Comment:
I have added a note about D2B. Currently there is no script for generating the D2B definition. This and it's description can be added later.